src/modules/fancyzones/editor/FancyZonesEditor/FancyZonesEditor.csproj

```diff
@@ -113,7 +105,12 @@
       <Generator>MSBuild:Compile</Generator>
       <SubType>Designer</SubType>
     </ApplicationDefinition>
+    <Compile Include="MonitorVM\Command.cs" />

```

Comment:
Typical naming conventions are:
/ViewModels (VMs)
/Views (xaml files, but leave yours where it is for now)
/Models (Class file that describe data, like MOnitorInfo.cs)
/Helpers (the command stuff)